cores/nRF5/wiring_analog_nRF52.c

```diff
@@ -88,10 +89,12 @@ void analogReference( eAnalogReference ulMode )
     case AR_INTERNAL:
     default:
       saadcReference = SAADC_CH_CONFIG_REFSEL_Internal;
+	  saadcGain      = SAADC_CH_CONFIG_GAIN_Gain1_5;
```

Comment:
A minor nit pick, the white space seems to be inconsistent here? Maybe tabs instead of spaces?